<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>crud_materiel</class>
 <widget class="QDialog" name="crud_materiel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>905</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion du Materiel</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">qApp-&gt;setStyleSheet(&quot;QLineEdit { background-color: blue }&quot;);</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>544</x>
     <y>240</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string>Nom Matériel :</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="nbr">
   <property name="geometry">
    <rect>
     <x>674</x>
     <y>300</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit#nbr {
    background-color: lightblue;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
}</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNoPredictiveText</set>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_supprimer_4">
   <property name="geometry">
    <rect>
     <x>744</x>
     <y>410</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#pb_supprimer_4 {
    background-color: lightblue;
    border-style: outset;
    border-width: 2px;
    border-radius: 6px;
    border-color: beige;
    font: bold 14px;
    min-width: 4em;
    padding: 6px;
}</string>
   </property>
   <property name="text">
    <string>Supprimer</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ref">
   <property name="geometry">
    <rect>
     <x>674</x>
     <y>180</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit#ref {
    background-color: lightblue;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
}</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNoPredictiveText</set>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>70</y>
     <width>301</width>
     <height>301</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(233,248,255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>tri par :</string>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>694</x>
     <y>110</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox#comboBox_2 {
    background-color: lightblue;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>544</x>
     <y>180</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string>Ref Matériel :</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="nom">
   <property name="geometry">
    <rect>
     <x>674</x>
     <y>240</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit#nom{
    background-color: lightblue;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
}</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNoPredictiveText</set>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_supprimer_3">
   <property name="geometry">
    <rect>
     <x>634</x>
     <y>410</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#pb_supprimer_3 {
    background-color: lightblue;
    border-style: outset;
    border-width: 2px;
    border-radius: 6px;
    border-color: beige;
    font: bold 14px;
    min-width: 4em;
    padding: 6px;
}</string>
   </property>
   <property name="text">
    <string>Modifier</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_ajouter">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>410</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#pb_ajouter {
    background-color: lightblue;
    border-style: outset;
    border-width: 2px;
    border-radius: 6px;
    border-color: beige;
    font: bold 14px;
    min-width: 4em;
    padding: 6px;
}</string>
   </property>
   <property name="text">
    <string>Ajouter</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>300</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string>Nombre :</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTableView" name="tabmateriel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>401</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>40</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox#comboBox {
    background-color: lightblue;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
}</string>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
   <property name="insertPolicy">
    <enum>QComboBox::NoInsert</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_id_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit#lineEdit_id_6 {
    background-color: lightblue;
    border-style: outset;
    border-width: 2px;
    border-color: beige;
}</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNoPredictiveText</set>
   </property>
   <property name="placeholderText">
    <string>Recherche par nom ou référence</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton0">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>10</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#pushButton0 {
    background-color: lightblue;
    border-style: outset;
    border-width: 2px;
    border-radius: 6px;
    border-color: beige;
    font: bold 14px;
    min-width: 4em;
    padding: 6px;
}</string>
   </property>
   <property name="text">
    <string>Retour</string>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>label_2</zorder>
  <zorder>nbr</zorder>
  <zorder>pb_supprimer_4</zorder>
  <zorder>ref</zorder>
  <zorder>comboBox_2</zorder>
  <zorder>label</zorder>
  <zorder>nom</zorder>
  <zorder>pb_supprimer_3</zorder>
  <zorder>pb_ajouter</zorder>
  <zorder>label_3</zorder>
  <zorder>tabmateriel</zorder>
  <zorder>comboBox</zorder>
  <zorder>lineEdit_id_6</zorder>
  <zorder>pushButton0</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
